tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -191,6 +195,81 @@ def _predict_proba(self, X, **kwargs):
         y_pred_proba = self.model.predict_proba(X)
         return self._convert_proba_to_unified_form(y_pred_proba)
 
+    def get_oof_pred_proba(self, X, normalize=None, **kwargs):
+        """X should be the same X passed to `.fit`"""
+        y_oof_pred_proba = self._get_oof_pred_proba(X=X, **kwargs)
+        if normalize is None:
+            normalize = self.normalize_pred_probas
+        if normalize:
+            y_oof_pred_proba = normalize_pred_probas(y_oof_pred_proba, self.problem_type)
+        y_oof_pred_proba = y_oof_pred_proba.astype(np.float32)
+        return y_oof_pred_proba
+
+    # FIXME: Unknown if this works with quantile regression
+    def _get_oof_pred_proba(self, X, y, **kwargs):
+        if self._daal:
+            raise AssertionError('DAAL forest backend does not support out-of-bag predictions.')
```

Comment:
These errors may be more helpful if they describe an alternative for the user.

Reply:
will plan to add more info when DAAL gets further along.